static/js/people.js

```diff
@@ -1,15 +1,15 @@
-"use strict";
+import md5 from "blueimp-md5";
+import _ from "lodash";
+import moment from "moment-timezone";
 
-const md5 = require("blueimp-md5");
-const _ = require("lodash");
-const moment = require("moment-timezone");
+import * as typeahead from "../shared/js/typeahead";
 
-require("unorm"); // String.prototype.normalize polyfill for IE11
-const typeahead = require("../shared/js/typeahead");
+import {FoldDict} from "./fold_dict.ts";
+import * as settings_data from "./settings_data";
+import * as util from "./util";
 
-const FoldDict = require("./fold_dict").FoldDict;
-const settings_data = require("./settings_data");
-const util = require("./util");
+// String.prototype.normalize polyfill for IE11
+require("unorm");
```

Comment:
This should also be converted to `import`.

Reply:
Ah, my script must have missed this!